server/metrics/metrics.go

```diff
@@ -158,91 +175,74 @@ func NewMetrics(info InstanceInfo) Metrics {
-		Name:        "change_event_total",
-		Help:        "The total number of MS Teams change events received.",
-		ConstLabels: additionalLabels,
-	}, []string{"change_type"})
-	m.registry.MustRegister(m.changeEventTotal)
-
-	m.processedChangeEventTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
-		Namespace:   MetricsNamespace,
-		Subsystem:   MetricsSubsystemEvents,
-		Name:        "processed_change_event_total",
+		Name:        "change_events_total",
 		Help:        "The total number of MS Teams change events processed.",
 		ConstLabels: additionalLabels,
 	}, []string{"change_type", "discarded_reason"})
-	m.registry.MustRegister(m.processedChangeEventTotal)
+	m.registry.MustRegister(m.changeEventsTotal)
 
-	m.lifecycleEventTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
+	m.lifecycleEventsTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace:   MetricsNamespace,
 		Subsystem:   MetricsSubsystemEvents,
-		Name:        "lifecycle_event_total",
+		Name:        "lifecycle_events_total",
 		Help:        "The total number of MS Teams lifecycle events received.",
 		ConstLabels: additionalLabels,
 	}, []string{"event_type"})
-	m.registry.MustRegister(m.lifecycleEventTotal)
+	m.registry.MustRegister(m.lifecycleEventsTotal)
 
-	m.messagesCount = prometheus.NewCounterVec(prometheus.CounterOpts{
+	m.messagesTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace:   MetricsNamespace,
 		Subsystem:   MetricsSubsystemEvents,
-		Name:        "messages_count",
+		Name:        "messages_total",
 		Help:        "The total number of messages for different actions and sources",
 		ConstLabels: additionalLabels,
 	}, []string{"action", "source", "is_direct"})
-	m.registry.MustRegister(m.messagesCount)
+	m.registry.MustRegister(m.messagesTotal)
 
-	m.reactionsCount = prometheus.NewCounterVec(prometheus.CounterOpts{
+	m.reactionsTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace:   MetricsNamespace,
 		Subsystem:   MetricsSubsystemEvents,
-		Name:        "reactions_count",
+		Name:        "reactions_total",
 		Help:        "The total number of reactions for different actions and sources",
 		ConstLabels: additionalLabels,
 	}, []string{"action", "source", "is_direct"})
-	m.registry.MustRegister(m.reactionsCount)
+	m.registry.MustRegister(m.reactionsTotal)
 
-	m.filesCount = prometheus.NewCounterVec(prometheus.CounterOpts{
+	m.filesTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace:   MetricsNamespace,
 		Subsystem:   MetricsSubsystemEvents,
-		Name:        "files_count",
+		Name:        "files_total",
 		Help:        "The total number of files for different actions and sources",
 		ConstLabels: additionalLabels,
 	}, []string{"action", "source", "is_direct", "discarded_reason"})
-	m.registry.MustRegister(m.filesCount)
+	m.registry.MustRegister(m.filesTotal)
 
-	m.messagesConfirmedCount = prometheus.NewCounterVec(prometheus.CounterOpts{
+	// TODO: Why?
```

Comment:
Left intact for now, but it's not clear if we need this over and above `messagesTotal` above.